src/tests/suite/SConscript

```diff
@@ -103,7 +103,8 @@ def scons():
                     daos_obj_array.c daos_obj.c daos_oid_alloc.c daos_pool.c
                     daos_rebuild.c daos_rebuild_common.c daos_rebuild_ec.c
                     daos_rebuild_simple.c daos_test.c daos_test_common.c
-                    daos_verify_consistency.c daos_degrade_ec.c""")
+                    daos_verify_consistency.c daos_aggregate_ec.c
+		    daos_degrade_ec.c""")
```

Comment:
is the indentation weird here or is that just a github artifact?

Reply:
i think just github:
https://github.com/daos-stack/daos/blob/c9f58852b2212f53b9fdccbdfcb2dc2a365fba6d/src/tests/suite/SConscript